### PR TITLE
Controllers features

### PR DESCRIPTION
**Pull request by mullenlucas**
## Implemented features:

- Added RSpec as a gem to your project.
- Created a Request spec file for all your controllers.
- Made checks for:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
     end
+
+    it 'renders correct template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes placeholder text' do
+      expect(response.body).to include(@user.name)
+    end
   end
```
Comment (Nemwel-Boniface) [inline spec/requests/posts_spec.rb]: - [x] For some reason, when I run the tests, some of them seem t be failing. Kindly look into why this is the case.

image

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -2,16 +2,41 @@
...
     end
+
+    it 'renders correct template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'includes placeholder text' do
+      expect(response.body).to include(@user.bio)
+    end
   end
```
Comment (Nemwel-Boniface) [inline spec/requests/users_spec.rb]: - [x] For some reason, when I run the tests, some of them seem t be failing. Kindly look into why this is the case.

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @mullenlucas     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
### Hello @mullenlucas 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------

**Pull request merged**